pontoon/base/models.py

```diff
@@ -531,6 +510,23 @@ def create_user_profile(sender, instance, created, **kwargs):
 post_save.connect(create_user_profile, sender=User)
 
 
+def get_translation_dict(entity, translations_dict, i=None):
```

Comment:
This should really be a method like `serialize` was. General rule of thumb is that if a function always takes a model instance as it's first argument, you should consider if it should be a method on the class instead.